### DISCUSSION TITLE
insufficient_quota

### DISCUSSION THREAD

**Discussion by notfifteen** [Zain-ul-din/whatsapp-ai-bot#19] [Q&A] [answered]
I just launched the bot and created the api key recently, but I get this error, does anyone know why? I have not used the apikey before.

```diff
-✖ CustomModel request fail | An error occur, at CustomModel.ts sendMessage(prompt, msg) err: Error: OpenAI error 429: {
-    "error": {
-        "message": "You exceeded your current quota, please check your plan and billing details.",
-        "type": "insufficient_quota",
-        "param": null,
-       "code": "insufficient_quota"
-    }
-}
```

**Comment by notfifteen** [comment, marked as answer]
It seems that OpenAI had limited me since I tried to use several times the same token, I have already fixed it, I apologize for opening this discussion without being patient.

**Reply by Zain-ul-din** [reply]
No problem!

A possible fixes is use your credit card to buy or another way is to create new account that i also did XD :-)

**Reply by getting55** [reply]
If you can use 5k USD per month, you can come to me to buy it, I can give you a discount, and if the amount is small, bind a credit card to recharge
